spec/requests/as_content_owner/content_owner_spec.rb

```diff
@@ -264,6 +265,32 @@
         it { expect(reference).not_to be }
       end
     end
+
+    describe '.upload_reference_file' do
+      let(:asset) { Yt::Asset.new id: ENV['YT_TEST_PARTNER_ASSET_ID'], auth: $content_owner }
+      let(:match_policy) { Yt::MatchPolicy.new asset_id: ENV['YT_TEST_PARTNER_ASSET_ID'], auth: $content_owner }
+
+      let(:upload_params) { {asset_id: asset.id, content_type: 'video'} }
+      let(:reference) { $content_owner.upload_reference_file path_or_url, upload_params }
+      after { reference.delete }
+
+      before do
+        asset.ownership.update(assetId: asset.id) && asset.ownership.obtain!
+        match_policy.update policy_id: ENV['YT_TEST_PARTNER_POLICY_ID']
+      end
+
+      context 'given the path to a local video file' do
+        let(:path_or_url) { File.expand_path '../reference_video.mp4', __FILE__ }
```

Comment:
This is a pretty big file (23.4MB). Is there something else we can do?

Reply:
A video needs to be at a minimum 20 seconds long in order to create a reference file, maybe I can compress and upload a lower quality version.

Reply:
Hrm, that's too bad. I wonder if we need to have an explicit test here for the local file case if we're just using `open-uri`...in a way we'd just be testing that open-uri works.

Reply:
Sure, I can change the test to use a remote URL instead.

Reply:
@dgb Forgot that I already had the remote URL test case here. Updated to remove local test case and file.